src/Microsoft.Health.Fhir.Api/Features/SMART/SmartClinicalScopesMiddleware.cs

```diff
@@ -111,8 +111,12 @@ public SmartClinicalScopesMiddleware(RequestDelegate next, ILogger<SmartClinical
                                 }
 
                                 fhirRequestContext.AccessControlContext.AllowedResourceActions.Add(new ScopeRestriction(resource, permittedDataActions, id));
-
-                                _logger.LogInformation("Resource and permitted data actions {Resource} {PermittedDataActions} ", resource, permittedDataActions);
+                                scopeRestrictions.Append(" ( ");
+                                scopeRestrictions.Append(resource);
+                                scopeRestrictions.Append(" - ");
+                                scopeRestrictions.Append(permittedDataActions);
+                                scopeRestrictions.Append(" ) ");
```

Comment:
nit: Sorry, one more comment - we could build this one one like below
scopeRestrictions.Append($" ( {resource}-{permittedDataActions} ) ");